config/database.yml

```diff
@@ -1,16 +1,18 @@
 default: &default
   adapter: mysql2
   encoding: utf8
-  username: collections_pub
-  password: collections_publisher
+  username: <%= ENV['DATABASE_USER'] || 'collections_pub' %>
+  password: <%= ENV['DATABASE_PASSWORD'] || 'collections_publisher' %>
```

Comment:
Do these need to be configurable? I don't think anything is using them and if I remember correctly DATABASE_URL takes precedence over them?